spec/lib/git-whistles/youtrack/branch_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
+require 'pry'
+require_relative '../../../../lib/git-whistles/youtrack/branch'
```

Comment:
im surprised you need to do this, can you add to the LOAD_PATH on the spec helper `lib` please? So then this should become `require git-whistles/...` same comment for all spec files